cmds/houndd/main.go

```diff
@@ -113,6 +116,18 @@ func runHttp(
 	return http.ListenAndServe(addr, m)
 }
 
+func getVersion() semver.Version {
+	ver, err := semver.Parse(version)
+	if err == nil {
+		return ver
+	}
+	return semver.Version{
```

Comment:
nit: The latest release is 0.3.0, technically. Are you seeing v1.0.0 listed somewhere?

Reply:
I will change it to 0.3.0